src/components/entity/filter/EntityFilterFacetPanel.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-expansion-panel>
+    <v-expansion-panel-header class="font-weight-bold">
+      <slot name="title">{{ title }}</slot>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content>
+      <EntityFilterFacets :filterField="filterField" :fields="fields" />
+      <template v-if="more.length">
+        <a
```

Comment:
Strongly avoid anchor tags unless they specifically link to another page. Instead opt for a div or span with class styling that makes it look like a link.